src/languages/arduino.js

```diff
@@ -94,7 +94,7 @@ function(hljs) {
         'DEFAULT OUTPUT INPUT HIGH LOW'
   };
 
-  var ARDUINO = hljs.getLanguage('cpp').rawDefinition();
+  var ARDUINO = hljs.requireLanguage('cpp').rawDefinition();
```

Comment:
Shouldn't this go to v.10 as a breaking change?

Reply:
Except it's not a breaking change.   Unless you're talking about using languages compiled for 9.17 with 9.16... that wouldn't work... but otherwise there is no breakage here... we're adding a new API, not renaming it.

Reply:
Ah, right. If you have no cpp, arduino won't work anyway (:

Reply:
My new build system resolves dependencies (so asking for `arduino` will always also get you `cpp`)... but still people requiring languages one at a time with NPM or something can get bitten by this... not sure how to prevent people from shooting themselves in the foot.